keyboards/keebwerk/nano_slider/keymaps/default/keymap.c

```diff
@@ -39,13 +40,13 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
     ),
     [_FN] = LAYOUT(
         TO(_DEMO),
```

Comment:
Target layer was renamed.

```suggestion
        TO(_MEDIA),
```